apps/linux/firefox.talon

```diff
@@ -1,9 +1,9 @@
 os: linux
 app: Firefox
+app: firefox
 -
-#todo tags
-# settings():
-	# tags = browser
+tag(): browser
+tag(): firefox
```

Comment:
I think this needs to be `user.firefox`, no?

Reply:
not for tags afaik. my vimvixen talon file triggers based on `tag: firefox` with this np.

i agree it's kinda weird though, as it differs from modes, settings, etc namespace